website/content/docs/upgrading/index.mdx

```diff
@@ -20,11 +20,25 @@ structure that make the data incompatible with a downgrade. If you need to roll
 back to a previous version of Vault, you should roll back your data store as
 well.
 
+Vault upgrades are designed such that large jumps (ie 1.3.10 -> 1.7.x) are
+supported. The upgrade notes for each intervening version must be reviewed. The
+upgrade notes may describe additional steps or configuration to update before,
+during, or after the upgrade.
+
 ## Learn
 
 Refer to the [Vault Upgrade Standard Procedure](https://learn.hashicorp.com/tutorials/vault/sop-upgrade)
 guide for step-by-step examples of upgrading Vault Enterprise.
 
+## Agent
+
+The Vault Agent is an API client of the Vault Server. Vault APIs are almost
+always backwards compatible. When they are not, this is called oout in the
+upgrade guide for the new Vault version, and there is a lengthy deprecation
+period. The Vault Agent version can lag behind the Vault Server version, though
+we recommend keeping all versions up to date with the most minor Vault version
```

Comment:
This is a little confusing, what does keeping up to date with the most minor Vault version mean?

Reply:
Fair. First of all I'm missing a word - "most recent minor Vault version". Does that make the sentence feel like it has enough value, or would you be interested in further re-tooling?

Reply:
```suggestion
we recommend keeping all Vault instances up to date with the most recent minor Vault version
```